fluster/decoders/chromium.py

```diff
@@ -0,0 +1,64 @@
+# Fluster - testing framework for decoders conformance
+# Copyright (C) 2021, Fluendo, S.A.
+#  Author: Manuel Jimeno Martínez <mjimeno@fluendo.com>, Fluendo, S.A.
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Library General Public
+# License as published by the Free Software Foundation; either
+# version 2 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Library General Public License for more details.
+#
+# You should have received a copy of the GNU Library General Public
+# License along with this library; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place - Suite 330,
+# Boston, MA 02111-1307, USA.
+# pylint: disable=import-outside-toplevel
+from abc import ABC
+
+from fluster.codec import Codec, OutputFormat
+from fluster.decoder import Decoder, register_decoder
+
+
+class ChromiumDecoder(Decoder, ABC):
+    """Generic class for Chromium decoder"""
+    description = "This is an implementation for the Chromium decoder"
+
+
+@register_decoder
+class ChromiumH264(ChromiumDecoder):
+    """H264 class for Chromium decoder"""
+    provider = 'Chromium'
+    codec = Codec.H264
+
+    def __init__(self) -> None:
+        super().__init__()
+        if not self.name:
+            self.name = f'{self.provider}-{self.codec.value}'
+        self.description = f'{self.provider} {self.codec.value} decoder for Chromium'
+
+    def decode(
+            self,
+            input_filepath: str,
+            output_filepath: str,
+            output_format: OutputFormat,
+            timeout: int,
+            verbose: bool,
+    ) -> str:
+        message_error = "Decoder doesn't available, module flusterchromium not installed"
```

Comment:
replace "doesn't" with "isn't"

Reply:
```suggestion
        message_error = "Decoder is not't available, module fluster-chromium not installed"
```

Reply:
we use fluster_chromiun instead of fluster-chromium